README.md

```diff
@@ -1,14 +1,15 @@
-# napari-allencell-segmenter
+# napari-aicssegmentation
 
-# This project has just started. It is not yet in a usable state.
+[![License](https://img.shields.io/pypi/l/napari-aicssegmentation.svg?color=green)](https://github.com/AllenCell/napari-aicssegmentation/raw/main/LICENSE)
+[![PyPI](https://img.shields.io/pypi/v/napari-aicssegmentation.svg?color=green)](https://pypi.org/project/napari-aicssegmentation)
+[![Python Version](https://img.shields.io/pypi/pyversions/napari-aicssegmentation.svg?color=green)](https://python.org)
+[![tests](https://github.com/AllenCell/napari-aicssegmentation/workflows/tests/badge.svg)](https://github.com/AllenCell/napari-aicssegmentation/actions)
+[![codecov](https://codecov.io/gh/AllenCell/napari-aicssegmentation/branch/main/graph/badge.svg)](https://codecov.io/gh/AllenCell/napari-aicssegmentation)
 
-[![License](https://img.shields.io/pypi/l/napari-allencell-segmenter.svg?color=green)](https://github.com/AllenCell/napari-allencell-segmenter/raw/main/LICENSE)
-[![PyPI](https://img.shields.io/pypi/v/napari-allencell-segmenter.svg?color=green)](https://pypi.org/project/napari-allencell-segmenter)
-[![Python Version](https://img.shields.io/pypi/pyversions/napari-allencell-segmenter.svg?color=green)](https://python.org)
-[![tests](https://github.com/AllenCell/napari-allencell-segmenter/workflows/tests/badge.svg)](https://github.com/AllenCell/napari-allencell-segmenter/actions)
-[![codecov](https://codecov.io/gh/AllenCell/napari-allencell-segmenter/branch/main/graph/badge.svg)](https://codecov.io/gh/AllenCell/napari-allencell-segmenter)
+A plugin that enables 3D image segmentation provided by Allen Institute for Cell Science
+
+The Allen Cell Structure Segmenter plugin for napari provides an intuitive graphical user interface to access the powerful segmentation capabilities of an open source 3D segmentation software package developed and maintained by the Allen Institute for Cell Science (classic workflows only with v1.0). ​The Allen Cell Structure Segmenter is a Python-based open source toolkit developed at the Allen Institute for Cell Science for 3D segmentation of intracellular structures in fluorescence microscope images. This toolkit brings together classic image segmentation and iterative deep learning workflows first to generate initial high-quality 3D intracellular structure segmentations and then to easily curate these results to generate the ground truths for building robust and accurate deep learning models. The toolkit takes advantage of the high replicate 3D live cell image data collected at the Allen Institute for Cell Science of over 30 endogenous fluorescently tagged human induced pluripotent stem cell (hiPSC) lines. Each cell line represents a different intracellular structure with one or more distinct localization patterns within undifferentiated hiPS cells and hiPSC-derived cardiomyocytes.
```

Comment:
`The Allen Cell Structure Segmenter plugin...` should be `The Allen Cell & Structure Segmenter plugin..`